### DISCUSSION TITLE
Current consumption for export meter for current day?

### DISCUSSION THREAD

**Discussion by srobertson92** [BottlecapDave/HomeAssistant-OctopusEnergy#222] [Ideas]
With a home mini, we get the current consumption sensor that allows you to add it to the energy dashboard, is there any capability for this with the export meters? Currently only gives previous day, but no real time or current consumption sensor.

**Comment by BottlecapDave** [comment]
The Octopus Home mini doesn't provide this data in the required unit of measurement. You can determines if it's exporting via the current demand sensor, but the data for current consumption doesn't seem to go negative. Full details of this discussion can be found at https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/198

**Comment by DuffHarry** [comment]
Hi @BottlecapDave - I'm new to home assistant, octopus energy and python. 
On the https://developer.octopus.energy/docs/api/#consumption site it mentions that half hour consumption can be brought in.  
I'm just checking if is this only for historic data or if this is actually for live data and the code here isn't updated for it yet.  Cheers Dave

**Reply by BottlecapDave** [reply]
Hi @DuffHarry - from the readme

> By default, it's not possible to include current consumption sensors. This is due to Octopus Energy only receive data from the smart meters up to the previous day. 

The data is available in half hour chunks (which is available as an attribute on the previous consumption sensors), but not for the current day
